src/disableActEnvironment.ts

```diff
@@ -0,0 +1,171 @@
+import {getConfig} from '@testing-library/dom'
+
+const dispose: typeof Symbol.dispose =
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  Symbol.dispose ?? Symbol.for('nodejs.dispose')
+
+export interface DisableActEnvironmentOptions {
```

Comment:
Should these options be mentioned in the README?

Reply:
I would honestly not - there's a bunch of other, probably less used, options I don't mention in the README.

I see the README as the tool to quickly pick it up and JSDoc as API documentation here.